apiary.apib

```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
+| 1                                        | 0              | Recommended Facebook review           |
+| 0                                        | 0              | Non Recommended Facebook review       |
+| Does not have  any value (Null or Blank) | 1-5            | Facebook review with non-zero rating. |
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Error Codes</caption>
+        <thead>
+        <tr>
+            <th>HTTP Status</th>
+            <th>BirdEye error code</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1161</td>
+            <td>Invalid API key</td>
+        </tr>
+        <tr>
+            <td>429 Too Many Requests</td>
+            <td>89</td>
+            <td>Rate limit exceeded</td>
+        </tr>
+        <tr>
+            <td>404 Not Found</td>
+            <td>1011</td>
+            <td>Business id is invalid</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1163</td>
+            <td>Invalid start index value</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1164</td>
+            <td>Invalid result count value</td>
+        </tr>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1167</td>
+            <td>API key is missing</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1186</td>
+            <td>Invalid from date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1187</td>
+            <td>Invalid to date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1227</td>
+            <td>Invalid updated from date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1228</td>
+            <td>Invalid updated to date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+## Get [/v1/review/report/count-by-rating/employee/{BusinessId}{?api_key}]
+Business review(s) from BirdEye platform can be fetched using following filter in request payload:
+
++ Parameters
 
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews Count By Driver [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+
+    curl -XPOST -H "Content-type: application/json" -d '{"businessNumbers": [145308764185002,145308769721320],"sourceAlias": ["google","citysearch"],"employees": [{"firstName": "String","lastName": "String","phone": "String"}],"fromDate": "04/01/2013","toDate": "05/3/2015"}' 'https://api.birdeye.com/resources/v1/review/report/count-by-rating/employee/145308769721320?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451'
```

Comment:
Should be like curl -H "content-type: application/json" -H "accept:application/json" -X POST to keep uniformity with existing documentation.

Reply:
Done